runtime/mux.go

```diff
@@ -14,6 +14,31 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
+// UnescapingMode defines the behavior of grpc-gateway for URL escaping.
```

Comment:
Nit:
```suggestion
// UnescapingMode defines the behavior of the grpc-gateway for URL escaping.
```

Reply:
I'll change grpc-gateway to ServeMux, since we're talking about a ServeMux option.